src/hooks.server.ts

```diff
@@ -1,6 +1,6 @@
 import { locales } from "$i18n";
 import type { Handle } from "@sveltejs/kit";
-import { unwrapFunctionStore, locale } from "svelte-i18n";
+import { locale, unwrapFunctionStore } from "svelte-i18n";
```

Comment:
Unrelated.

Reply:
i think the auto-formatter made this change, i can revert it if need be.